### PR TITLE
Update copyright

### PR DESCRIPTION
**Pull request by ThibaultMarzullo**
Update the copyright statement on all files to cover 2022.

### CODE REVIEW

**Comment by nllong** [comment]
looks great! Not sure why the CI/Model Articulation tests are failing. Can you take a look?

**Comment by nllong** [comment]
> looks great! Not sure why the CI/Model Articulation tests are failing. Can you take a look?

Oh wait, it is the year thingy: https://github.com/BuildingSync/BuildingSync-gem/runs/5279348841?check_suite_focus=true#step:13:149

**Comment by ThibaultMarzullo** [comment]
Yeah, I just noticed that. I will change the .osw file and commit the changes!

File: spec/files/filecomparison/originalfiles/in.idf [hunk 1]
```diff
@@ -2475,10 +2475,10 @@ RunPeriod,
   Run Period 1,                           !- Name
   1,                                      !- Begin Month
   1,                                      !- Begin Day of Month
-  2021,                                   !- Begin Year
+  2022,                                   !- Begin Year
```
Comment (nllong) [inline spec/files/filecomparison/originalfiles/in.idf:2478]: 📅

**Review by nllong — APPROVED** [review]
bussin'

**Pull request merged**